### PR TITLE
Raspberrypi

### PR DESCRIPTION
**Pull request by KairaMiu**

### CODE REVIEW

**Review by KlausPopp — CHANGES_REQUESTED** [review]
dobi build of raspberrypi4-64-edgefarm-devtools-build-image fails:

| + mender-artifact write rootfs-image -n raspberrypi4-64-edgefarm-devtools-grep: /git/raspberrypi4-64-edgefarm-devtools-image/layer-revs: No such file or directory

Ci pipeline failed due to fetching failures

File: README.md [hunk 1]
```diff
@@ -140,3 +141,17 @@ Start deployment
 ```bash
 ./dobi.sh cpu01-edgefarm-minio-deploy
 ```
+
+### Install Raspberry Pi sdimg on SD Card
+
+Find build images:
+
+```
+ls */install/images/raspberrypi4-64/core-image-base-raspberrypi4-64-*.sdimg
```
Comment (KlausPopp) [inline README.md:150]: The filename is something like EdgeFarm-Devtools-Image-raspberrypi4-64-20220224142829.sdimg

File: README.md [hunk 2]
```diff
@@ -140,3 +141,17 @@ Start deployment
...
+```
+
+Install on SD Card:
+
+```
+sudo dd if=<PATH-TO-IMAGE>.sdimg of=<DEVICE> bs=1M && sudo sync
```
Comment (KlausPopp) [inline README.md:156]: Or on windows, use https://www.balena.io/etcher/

Add some notes about boot behaviour: 
- Kernel boot messages appear on serial console (is this a good idea?, dummy users would expect it on HDMI)
- ssh is enabled, locate device in network, look for  device named "raspberrypi4-64"
- login with ssh root@<ip>

**Review by KlausPopp — CHANGES_REQUESTED** [review]

File: dobi.yaml [hunk 3]
```diff
@@ -97,51 +97,103 @@ job=mender-accept-latest-auth:
...
   tasks:
-    - cpu01-base-build-image
+    # - cpu01-base-build-image
+    # - cpu01-devtools-build-image
     - cpu01-edgefarm-build-image
-    - cpu01-devtools-build-image
+    - cpu01-edgefarm-devtools-build-image
+    - raspberrypi4-64-edgefarm-build-image
+    - raspberrypi4-64-edgefarm-devtools-build-image
+    - qemux86-64-edgefarm-devtools-build-image
```
[the remark this answers is not in the document]
Comment (KairaMiu) [reply dobi.yaml]: Done

**Comment by KairaMiu** [comment]
> dobi build of raspberrypi4-64-edgefarm-devtools-build-image fails:
> 
> | + mender-artifact write rootfs-image -n raspberrypi4-64-edgefarm-devtools-grep: /git/raspberrypi4-64-edgefarm-devtools-image/layer-revs: No such file or directory
> 
> Ci pipeline failed due to fetching failures

Fixed.

File: README.md (hunk 2 above)
Comment (KairaMiu) [reply README.md:156]: Added instructions how to install with Raspberry Pi Imager (Windows / Linux)
Login instructions not part of section `Install Images`

**Review by siredmar — APPROVED** [review]
lgtm

**Pull request merged**